preset/rabbitmq/preset.go

```diff
@@ -22,6 +26,13 @@ const defaultVersion = "alpine"
 const defaultPort = 5672
 const managementPort = 15672
 
+// Message is a single message sent to RabbitMQ.
+type Message struct {
+	Queue       string `json:"queue"`
+	ContentType string `json:"contentType"`
+	Body        string `json:"body"`
```

Comment:
I agree with your suggestion to support both binary and string messages, just like in memcached preset, that's a cool idea.

Although we can do that in the future in a separate unrelated pull request (and you don't have to work on it of course), I think we should rename `Body` into something like `StringBody`, to allow adding a regular `Body` (`[]byte` type) later. I think `[]byte` should be called `Body` because it's the default in rabbitmq client, and `StringBody` is a nice utility wrapper that we add for convenience.